lua/astrocommunity/pack/cpp/cpp.lua

```diff
@@ -0,0 +1,19 @@
+local utils = require "astronvim.utils"
+return {
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if opts.ensure_installed ~= "all" then
+        opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, { "cpp", "c" })
+      end
+    end,
+  },
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "clangd") end,
+  },
+  {
+    "jay-babu/mason-null-ls.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "clang-format") end,
+  },
```

Comment:
```suggestion
  },
  {"p00f/clangd_extensions.nvim",
      ft = { "c", "cpp", "objc", "objcpp", "h", "hpp" },
  }, { "Civitasv/cmake-tools.nvim",
      config = function()
        require("user.cle").cmake_config()
      end,
      ft = { "c", "cpp", "objc", "objcpp", "h", "hpp" },}
```

Reply:
I think the default is that the server is not setup. So in theory we wouldn't need that, no?

Reply:
yeah and

```lua
init = function() table.insert(astronvim.lsp.skip_setup, "tsserver") end
```

Reply:
Ah got it. I just stole from my old config 😥

Reply:
> yeah and
> 
> 
> 
> ```lua
> 
> init = function() table.insert(astronvim.lsp.skip_setup, "clangd") end
> 
> ```

why `table.insert` over `list_insert_unique`?

Reply:
This is a good point. I changed it